modules/integration/tests-integration/lazy-loading-test/src/test/java/org/wso2/appserver/integration/lazy/loading/LazyLoadingBaseTest.java

```diff
@@ -0,0 +1,486 @@
+    private static final long DEPLOYMENT_DELAY_IN_MILLISECONDS = 90 * 1000;
+    protected static final int CONCURRENT_THREAD_COUNT = 40;
+    private static final String CARBON_XML = "carbon.xml";
+    private static final String CARBON_REPOSITORY_LOCATION =
+            CARBON_HOME + File.separator + "repository" + File.separator + "conf" + File.separator + CARBON_XML;
+    private static final Log log = LogFactory.getLog(LazyLoadingBaseTest.class);
+    private long webAppIdleTime;
+    private long tenantIdleTime;
+    protected String superTenantDomain;
+    protected String tenantDomain1;
+    protected String tenantDomain2;
+    protected WebAppAdminClient webAppAdminClient;
+    protected ServerConfigurationManager serverManager;
+    protected String hostURL;
+    protected String artifactsLocation;
+
+
+    @Override
+    /**
+     * Login as super admin and do the replacement of carbon.xml with Ghost deployment
+     * configuration and deployment of the tenant-info-service web application.
+     * At the end it will restart the server gracefully.
+     * @throws Exception - Throws if initialisation fails.
+     */
+    public void init() throws Exception {
+        super.init();
+        serverManager = new ServerConfigurationManager(asServer);
+        webAppAdminClient = new WebAppAdminClient(backendURL, sessionCookie);
+        tenantIdleTime =
+                Long.parseLong(asServer.getConfigurationNode(TENANT_IDLE_XPATH).getNodeValue()) * 60 * 1000;
+        webAppIdleTime =
+                Long.parseLong(asServer.getConfigurationNode(WEB_APP_IDLE_XPATH).getNodeValue()) * 60 * 1000;
+        artifactsLocation =
+                TestConfigurationProvider.getResourceLocation() + File.separator + "artifacts" + File.separator +
+                        "AS" + File.separator + "ghost" + File.separator;
+        String tenantInfoServiceArtifactLocation =
+                artifactsLocation + TENANT_INFO_SERVICE_FILE_NAME;
+        String carbonArtifactLocation = artifactsLocation + CARBON_XML;
+        File sourceFile = new File(carbonArtifactLocation);
+        File targetFile = new File(CARBON_REPOSITORY_LOCATION);
+        superTenantDomain =
+                new AutomationContext(PRODUCT_GROUP_NAME, INSTANCE_NAME, SUPER_TENANT_DOMAIN_KEY, ADMIN_USER_KEY).
+                        getSuperTenant().getDomain();
+        tenantDomain1 =
+                new AutomationContext(PRODUCT_GROUP_NAME, INSTANCE_NAME, TENANT_DOMAIN_1_KEY, ADMIN_USER_KEY).
+                        getContextTenant().getDomain();
+        tenantDomain2 =
+                new AutomationContext(PRODUCT_GROUP_NAME, INSTANCE_NAME, TENANT_DOMAIN_2_KEY, ADMIN_USER_KEY).
+                        getContextTenant().getDomain();
+        serverManager.applyConfigurationWithoutRestart(sourceFile, targetFile, true);
+        log.info("carbon.xml replaced with :" + carbonArtifactLocation);
+        webAppAdminClient.warFileUplaoder(tenantInfoServiceArtifactLocation);
```

Comment:
@prabathabey - +1, we need to change the method name to "uploadWarFile()" or something. This class is admin service client written very long ago. So changing the name required many test case level modification. I will get automation team member to refactor the code and fix this properly.